swap_meet/vendor.py

```diff
@@ -0,0 +1,77 @@
+
+class Vendor:
+
+    def __init__(self, inventory= None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+        
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
You don't need the `else:`

```suggestion
        return False
```